migcontrol/templates/base.html

```diff
@@ -105,7 +105,10 @@
         </ul>
         <form class="search-form d-flex" action="{% url 'search' %}" method="GET">
           <input name="query" class="form-control me-2" type="search" placeholder="{% trans "Search" %}" aria-label="Search">
-          <button class="btn btn-success" type="submit">{% trans "Search" %}</button>
+
+
+          <button class="fa fa-search" style="color:red" type="submit" ></button>
```

Comment:
Hi Tchili, you have added `style="color:red"` which is an inline style. We cannot and should not use this. Can you remove it? Thanks.